Makefile

```diff
@@ -0,0 +1,43 @@
+PERL5LIB_:=../..:os-autoinst:lib:tests/installation:tests/x11:tests/qa_automation:tests/virt_autotest:tests/cpu_bugs:$$PERL5LIB
+
+.PHONY: all
+all:
+
+.PHONY: help
+help:
+	echo "Call 'make test' to call tests"
+
+.PHONY: prepare
+prepare:
+	git clone https://github.com/os-autoinst/os-autoinst.git
+	$(MAKE) check-links
+	cd os-autoinst && cpanm -nq --installdeps .
+	cpanm -nq --installdeps .
+
+os-autoinst/:
+	@test -d os-autoinst || (echo "Missing test requirements, \
+link a local working copy of 'os-autoinst' into this \
+folder or call 'make prepare' to install download a copy necessary for \
+testing" && exit 2)
+
+tools/tidy: os-autoinst/
+	@test -e tools/tidy || ln -s ../os-autoinst/tools/tidy tools/
+	@test -e tools/absolutize || ln -s ../os-autoinst/tools/absolutize tools/
+	@test -e .perltidyrc || ln -s os-autoinst/.perltidyrc ./
+
+tools/lib/: os-autoinst/
+	@test -e tools/lib || ln -s ../os-autoinst/tools/lib tools/
+
+.PHONY: check-links
+check-links: tools/tidy tools/lib/ os-autoinst/
+
+.PHONY: test-compile
+test-compile: check-links
+	export PERL5LIB=${PERL5LIB_} ; ( git ls-files "*.pm" || find . -name \*.pm|grep -v /os-autoinst/ ) | parallel perl -c 2>&1 | grep -v " OK$$" && exit 2; true
```

Comment:
hm, as we already depend on os-autoinst I think we can include that in a helper in os-autoinst and use from there.